### ISSUE TITLE
The option 'services.sshd' does not exist...

### ISSUE THREAD

**Issue by DrVoidest** [numtide/system-manager#130]
**Describe the bug**
I am trying to configure ssh using system-manager and can't seem to find where you put it. I have tried under config, evironment, etc, and systemd.services. Since ssh stores its config under /etc/ssh/sshd_config I assume its supported.

**To Reproduce**

Put anywhere in the config `services.sshd.enable = true;`

**Expected behavior**
Enables an open ssh server.

**System information**

Ubuntu 24.04 - fresh install

**Additional context**

I would like to use it to replace old versions of ssh on legacy systems then set the ssh config. I am only testing this on Ubuntu now though.

**Comment by r-vdp** [comment]
Yeah, we need to include the ssh module from nixpkgs, or write a custom one, But neither is trivial currently.
I'd be happy to review a PR for this though, or I might get to it at some point.

**Comment by DrVoidest** [comment]
I'm an inexperienced dev, but I will try taking a wack at it.

**Comment by jfroche** [comment]
Resolved by https://github.com/numtide/system-manager/pull/422 (openssh daemon module).

Closing — please reopen if the new module does not cover your use case.
